projects/media-viewer/src/assets/sass/angular-tree-component.scss

```diff
@@ -0,0 +1,84 @@
+.tree-children.tree-children-no-padding { padding-left: 0 }
+.tree-children { padding-left: 20px; overflow: hidden }
```

Comment:
not sure if worth  turning this into the scss style rather then css

Reply:
there's quite a few of them, and they belong to the external library, so I'm inclined not to